src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -47,132 +54,187 @@ <h4 class="panel-title border-bottom-small" translate>
                 <label>
                   <input name="suggestedAmount"
-                         type="radio"
-                         ng-checked="$ctrl.customInputActive" />
-                  <input class="form-control form-control-subtle"
-                         name="amount"
-                         type="text"
-                         step="1"
-                         tabindex="-1"
-                         ng-model="$ctrl.customAmount"
-                         ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
-                         ng-focus="$ctrl.customInputActive = true;"
-                         ng-required="$ctrl.customInputActive"
-                         placeholder="{{'OTHER_PLACEHOLDER' | translate}}" />
+                        type="radio"
+                        ng-click="$ctrl.changeAmount(suggested.amount)"
+                        ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
+                  {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
                 </label>
-                <div role="alert" ng-messages="$ctrl.itemConfigForm.amount.$error"
-                     ng-if="($ctrl.itemConfigForm.amount | showErrors)">
-                  <div class="help-block" ng-message="pattern" translate>{{'VALID_DOLLAR_AMOUNT_ERROR'}}</div>
-                  <div class="help-block" ng-message="required" translate>{{'AMOUNT_EMPTY_ERROR'}}</div>
-                  <div class="help-block" ng-message="minimum" translate translate-values="{currencyLimit: (1 | currency)}">{{'AMOUNT_MIN_ERROR'}}</div>
-                  <div class="help-block" ng-message="maximum" translate translate-values="{currencyLimit: (10000000 | currency)}">
-                    {{'AMOUNT_MAX_ERROR'}}
+              </div>
+              <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
+                <div class="form-group">
+                  <label>
+                    <input name="suggestedAmount"
+                          type="radio"
+                          ng-checked="$ctrl.customInputActive" />
+                    <input class="form-control form-control-subtle"
+                          name="amount"
+                          type="text"
+                          step="1"
+                          tabindex="-1"
+                          ng-model="$ctrl.customAmount"
+                          ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
+                          ng-focus="$ctrl.customInputActive = true;"
+                          ng-required="$ctrl.customInputActive"
+                          placeholder="{{'OTHER_PLACEHOLDER' | translate}}" />
+                  </label>
+                  <div role="alert" ng-messages="$ctrl.itemConfigForm.amount.$error"
+                      ng-if="($ctrl.itemConfigForm.amount | showErrors)">
```

Comment:
```suggestion
                      ng-focus="$ctrl.customInputActive = true"
```

Reply:
Not sure how this happened, but the suggestion I meant to make was removing the semicolon in `ng-focus`: `ng-focus="$ctrl.customInputActive = true;"` --> `ng-focus="$ctrl.customInputActive = true"`. I think GitHub got confused about which line I was commenting on. Sorry about that! You can put this back to how it was.